dallinger/frontend/static/scripts/dallinger2.js

```diff
@@ -107,12 +107,53 @@ var dallinger = (function () {
       type: 'json',
       success: function (resp) { deferred.resolve(resp); },
       error: function (err) {
+        var $form, errorResponse, request_data, worker_id, hit_id, assignment_id;
```

Comment:
This felt like a lot to stuff into the error handler, especially with all raw details exposed. Even 1 or 2 named helper functions would have helped my brain though it (`addToForm('participant_id', data.participant_id)`)?)

Reply:
Yeah, an `add_hidden_input($form, name, value)` and function would probably make things much clearer. I'll add that.